### DISCUSSION TITLE
NDI with tiktok Live Studio

### DISCUSSION THREAD

**Discussion by ditaeon** [DistroAV/DistroAV#847] [Q&A - Support]
Hi everyone, new to the world of NDI and was wondering,

Can I get NDI to work with Tiktok Live studio?

**Comment by OmarVargasEngnr** [comment]
Hi
The way that I found it is use OBS with a NDI plugin and launch virtual camera

**Comment by Trouffman** [comment]
Use the NDI Webcam software (part of the NDI Tools package).

TikTok studio : select NDI webcam 1
And in NDI webcam, select your NDI feed.

**Reply by arcbolt-tech** [reply]
> Use the NDI Webcam software (part of the NDI Tools package).
> 
> TikTok studio : select NDI webcam 1 And in NDI webcam, select your NDI feed.

Does that Webcam software also send the audio, too?

**Reply by Trouffman** [reply]
Yes
